client/src/components/backgroundImage/BackgroundImage.jsx

```diff
@@ -9,4 +10,8 @@ function BackgroundImage({ bgImage }) {
   );
 }
 
+BackgroundImage.propTypes = {
+  bgImage: backgroundImageBgImageType
```

Comment:
This is BackgroundImage component's scope. It would be cleaner if your prop name `bgImage` would be, say, just `src`